QuantumLink/Common/Networking/Protocol/Requests/AuthRequest.cs

```diff
@@ -1,33 +1,30 @@
-﻿using QuantumLink.networking.protocol;
-
-namespace Common.networking.protocol.requests
+﻿namespace QuantumLink.Common.Networking.Protocol.Requests
 {
     public sealed class AuthRequest
     {
         public sealed class AuthRequestEncoder : OutboundPacket
         {
             public AuthRequestEncoder(AuthRequest request) : base(0)
             {
-                this.writer.Write(request.Username);
-                this.writer.Write(request.Password);
-                this.writer.Write(request.CreateAccount);
+                this.Writer.Write(request.Username);
+                this.Writer.Write(request.Password);
+                this.Writer.Write(request.CreateAccount);
             }
         }
 
         public sealed class AuthRequestDecoder : InboundPacket
         {
-            public readonly AuthRequest DecodedAuthRequest;
+            public AuthRequest DecodedAuthRequest { get; }
 
             public AuthRequestDecoder(InboundPacket inboundPacket) : base(inboundPacket)
             {
-                this.DecodedAuthRequest = new AuthRequest(this.reader.ReadString(), this.reader.ReadString(), this.reader.ReadBoolean());
+                this.DecodedAuthRequest = new AuthRequest(this.Reader.ReadString(), this.Reader.ReadString(), this.Reader.ReadBoolean());
             }
         }
 
-        public readonly string Username;
-        public readonly string Password;
-
-        public readonly bool CreateAccount;
+        public string Username      { get; }
+        public string Password      { get; }
+        public bool   CreateAccount { get; }

```

Comment:
Why the spacing in between types and identifiers

Reply:
IDE setting I left on, my apologies